src/GetGeolocation/GetGeolocation.js

```diff
@@ -0,0 +1,25 @@
+export class GetGeolocation extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      position: 'Click me to get location'
+    };
+  }
+  getLocation = () => {
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(this.showPosition);
+    } else {
+      this.setState({ position: 'Geolocation is not supported by this browser' });
+    }
+  }
+  showPosition = (position) => {
+    this.setState({ position: `Latitude:  ${position.coords.latitude}  Longitude: ${position.coords.longitude}` });
+  }
+
+  clickHandler = () => {
+    this.getLocation();
```

Comment:
why you need extra mettod `getLocation`?

Reply:
removed